recipes/geolite2pp/all/patches/0001-fix-cmake.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/CM_definitions.cmake b/CM_definitions.cmake
+index dfb24d7..9f6ecb5 100644
+--- a/CM_definitions.cmake
++++ b/CM_definitions.cmake
+@@ -11,15 +11,15 @@ ELSE ()
+ ENDIF ()
+ 
+ ADD_DEFINITIONS ( "-std=gnu++11" )
```

Comment:
Maybe we can also comment this out and add it to the recipe? If it's too much work, don't do it, just a nice to have

Reply:
@AbrilRBS 
Thank you for your review!
I fixed it. Please review it again.

I am not yet confident about how to specify the C++ version in a recipe.